### PR TITLE
f-alert@v5.0.1 - Read alert messages to accessible users

### PR DESCRIPTION
**Pull request by tommcclean**
- Alert messages (success and danger) are now read out when displayed on screen.
- This only works if the alert is rendered using `v-show` in the parent component, as `aria-live` needs to be present on DOM load, `v-if` excludes the markup from the DOM

### CODE REVIEW

File: packages/components/molecules/f-alert/src/components/_tests/Alert.test.js [hunk 1]
```diff
@@ -164,4 +164,42 @@ describe('Alert', () => {
             });
         });
     });
+
+    describe('ariaLiveAttribute', () => {
```
Comment (jamieomaguire) [inline packages/components/molecules/f-alert/src/components/_tests/Alert.test.js:168]: do we want to test for the presence of the actual attribute in the html as well?

Comment (tommcclean) [reply packages/components/molecules/f-alert/src/components/_tests/Alert.test.js:168]: I imagine if the attribute was missing both tests would fail

Comment (tommcclean) [reply packages/components/molecules/f-alert/src/components/_tests/Alert.test.js:168]: I did go down the component test route, but it didn't look obvious how to provide different props, ill have another look and see.

Comment (tommcclean) [reply packages/components/molecules/f-alert/src/components/_tests/Alert.test.js:168]: Added a test to ensure the attribute is present

**Review by jamieomaguire — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**